service/src/main/java/bio/terra/workspace/app/controller/ReferencedGcpResourceController.java

```diff
@@ -601,8 +644,15 @@ public ResponseEntity<Void> deleteDataRepoSnapshotReference(UUID workspaceUuid,
     AuthenticatedUserRequest userRequest = getAuthenticatedInfo();
     workspaceService.validateWorkspaceAndAction(
         userRequest, workspaceUuid, SamWorkspaceAction.DELETE_REFERENCE);
+    var userStatusInfo =
```

Comment:
This is the fifth time for the same code. I think it should either move into the service or be a method in this class.